ingestion-data/staging/dataset-config/la-fires-planet.json

```diff
@@ -0,0 +1,95 @@
+{
+  "license": "CC0-1.0",
+  "stac_version": "1.0.0",
+  "links": [],
+  "spatial_extent": {
+    "xmin": -118.357,
+    "ymin": 33.887,
+    "xmax": -117.895,
+    "ymax": 32.279
+  },
+  "temporal_extent": {
+    "startdate": "2025-01-09T00:00:00Z",
+    "enddate": "2025-01-09T23:59:59Z"
+  },
+  "discovery_items": [
+    {
+      "upload": false,
+      "cogify": false,
+      "dry_run": false,
+      "filename_regex": "(.*)LA_Fires_Planet_RGB_(.*).tif$",
+      "use_multithreading": false,
+      "discovery": "s3",
+      "prefix": "LA-Fires/",
+      "bucket": "veda-data-store-staging"
+    }
+  ],
+  "sample_files": [
+    "s3://veda-data-store-staging/LA-Fires/LA_Fires_Planet_RGB_1_cog_2025-01-09.tif",
+    "s3://veda-data-store-staging/LA-Fires/LA_Fires_Planet_RGB_2_cog_2025-01-09.tif",
+    "s3://veda-data-store-staging/LA-Fires/LA_Fires_Planet_RGB_3_cog_2025-01-09.tif"
+  ],
+  "data_type": "cog",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/render/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+  ],
+  "item_assets": {
+    "cog_default": {
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "roles": [
+        "data",
+        "layer"
+      ],
+      "title": "Default COG Layer",
+      "description": "Cloud optimized default layer to display on map"
+    }
+  },
+  "providers": [
+    {
+      "name": "NASA VEDA",
+      "roles": [
+        "host"
+      ],
+      "url": "https://www.earthdata.nasa.gov/dashboard/"
+    }
+  ],
+  "assets": {
+    "thumbnail": {
+      "title": "Thumbnail",
+      "type": "image/png",
+      "roles": [
+        "thumbnail"
+      ],
+      "description": "Planet Labs Commercial Satellite Imagery of the Eaton Fire (January 9, 2025).",
+      "href": "https://thumbnails.openveda.cloud/la-fires-thumbnail.png"
+    }
+  },
+  "collection": "la-fires-planet",
+  "title": "PlanetScope Satellite Imagery (2025 Eaton Fire)",
+  "description": "TrueColor RGB commercial satellite imagery from Planet Labs of the Eaton Fire on January 9, 2025 north of Los Angeles, CA. This data was made available through the NASA Commercial Satellite Data Acquisition (CSDA) Program",
+  "is_periodic": false,
+  "time_density": "day",
```

Comment:
```suggestion
  "dashboard:is_periodic": false,
```

Reply:
I think I had this property wrong in the template I have been using and shared. Sorry about that!